Library/Homebrew/cask/cask_loader.rb

```diff
@@ -588,8 +588,7 @@ def self.default_path(token)
     def self.find_cask_in_tap(token, tap)
       filename = "#{token}.rb"
 
-      Tap.cask_files_by_name(tap)
-         .fetch(token, tap.cask_dir/filename)
+      tap.cask_files_by_name.fetch(token, tap.cask_dir/filename)
```

Comment:
```suggestion
      tap.cask_files_by_name.fetch(token) { tap.cask_dir/filename }
```

Nit: You could use a block here to avoid building Pathname objects  when the result is in the hash.

Reply:
@apainintheneck Seems like a good follow-up PR if it has any measurable performance impact.

Reply:
I don't have any benchmarks to back this up but we have had a lot of performance problems related to creating loads of `Pathname` objects before. It just seemed like an easy way to avoid that.